robotiq_s_model_visualization/cfg/robotiq_hand.urdf.xacro

```diff
@@ -0,0 +1,798 @@
+<?xml version="1.0"?>
+<!--
+robotiq_hand_macro(prefix)
+
+Creates an articulated s-model hand with the passed in prefix.  If there
+is only one hand in the urdf then an empty prefix can be used "".  If
+there are multiple hands then a prefix followed by an "_" is needed.
+-->
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="robotiq_hand">
+
+  <xacro:macro name="robotiq_finger" params="prefix">
+    <!-- Link list -->
+
+    <!-- Link 0 -->
+    <link name="${prefix}link_0">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://robotiq_s_model_visualization/meshes/s-model_articulated/visual/link_0.STL" />
```

Comment:
We normally try to keep everything lowercase in ROS-Industrial pkgs. Files are a direct copy from drcsim pkg though.